draft-js/draft-js-tests-0.2.2.tsx

```diff
@@ -0,0 +1,183 @@
+/// <reference path="draft-js-0.2.2.d.ts" />
+/// <reference path="../react/react-global.d.ts" />
+/// <reference path="../react/react-dom.d.ts"/>
+
+import * as React from "react";
+
+import {
+Editor,
```

Comment:
hi @zhengbli, did I missed something?
Indentation for files are the same as they were.